frontend_tests/node_tests/templates.js

```diff
@@ -221,16 +221,17 @@ function render(template_name, args) {
     html += "</table>";
 
     var buttons = $(html).find('.button');
+    var is_admin = page_params.is_admin;
+    if (is_admin) {
```

Comment:
@showell The thing is that the code (asserting that the 3 buttons are present) inside this if statement is for only when `page_params.is_admin = true` . When it is false then these buttons are not present, so basically we have path to be exercised only in "true" case and no code to be tested in "false" case. So, in a way we are exercising both the paths, first by entering the if condition and second by not.